common/reviews/api/web-library-build.api.ts

```diff
@@ -198,7 +198,7 @@ class TypeScriptTask extends GulpTask<ITypeScriptTaskConfig> {
   // (undocumented)
   getCleanMatch(buildConfig: IBuildConfig, taskConfig?: ITypeScriptTaskConfig): string[];
   // (undocumented)
-  protected loadSchema(): Object;
+  loadSchema(): Object;
```

Comment:
These were from a different build where I bypassed the CI (due issues getting a build queued)